classes/04class/exercises/c04-iac02/mfreitassm/main.tfvars

```diff
@@ -0,0 +1,13 @@
+aws_region = "ap-southeast-2"
+
+cidr_block_vpc = "10.0.0.0/16"
+
+private_subnet_cidr_blocks = ["10.0.0.0/24", "10.0.10.0/24"]
+private_subnet_names       = ["private-a", "private-b"]
+
+
+public_subnet_cidr_blocks = ["10.0.1.0/24", "10.0.11.0/24"]
+public_subnet_names       = ["public-a", "public-b"]
```

Comment:
wouldn't be better to keep those values together as they are directly related?

you can define your own object in TF using `object`:
https://www.terraform.io/docs/configuration/variables.html#declaring-an-input-variable

Something like this:
```
variable "subnets" {
  type = list(object({
    name = string
    cidr = string
    is_public = boolean
  }))
}
```